.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Release Checklist
+about: Create a release checklist
+title: "[RELEASE]"
+labels: ''
+assignees: ''
+
+---
+
+- [ ] confirm works reasonably in the field someplace: @
+- [ ] confirm launcher build: @
+- [ ] confirm with manual runs of a stress test app: @
```

Comment:
We need to define what this going to be, but I suppose it's more of a smoke test than a stress test.
```suggestion
- [ ] confirm with manual runs of a test app: @
```